### ISSUE TITLE
Visual Studio 2010 Getting Speech Feedback on Cursor Movement Not Involving Arrow Keys

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2470]
**Reported by Kerneels on 2012-06-19 20:12**
For example, when issuing a command in VS2010 that moves the caret / cursor such as Ctrl+] for jumping to the matching brace, NVDA doesn't read the new line like what whould happen if you arrowed down / up / left / right. 

In order to hear the new line you have to do NVDA key + up arrow. However, Braille seems to refresh properly. 

Ideal behaviour would be to have NVDA read the whole line if the cursor moved to a new line, read the next character if the cursor stayed in the same line and simply moved one character to the left or right, and read the containing word if the cursor moved more than a character in the same word, or jumped a word boundary and landed on a new word.

**Comment by feerrenrut** [comment]
Happy to accept a PR for this, however given the age of this ticket and VS 2010 we won't implement this internally. Marking as priority 4.

**Comment by leonardder** [comment]
This hasn't received attention for the last two years. I"m closing this for now. If there is a similar request for a newer version of Visual Studio, it is probably best to create a new issue.
